### PR TITLE
feat: add webcal button

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/molecules/CalendarLink.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+const CalendarLink: React.FC<CalendarLinkProps> = ({ id }) => {
+  const reference = useRef<HTMLDivElement>(null);
+  const handleClick = () => setMenuShown(!menuShown);
+  const [menuShown, setMenuShown] = useState(false);
+  useEffect(() => {
+    setMenuShown(false);
+  }, []);
+
+  useEffect(() => {
```
Comment (jeysal) [inline packages/react-components/src/molecules/CalendarLink.tsx:92]: Wow, with all other things, like the user menu, we just got away without implementing this ^^

File: packages/react-components/src/molecules/CalendarLink.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                </div>
+              </Anchor>
+            </li>
+            <li>
+              <Anchor href={webcal.toString()}>
+                <div css={calendarLinkStyles}>{outlookIcon} Add to Outlook</div>
+              </Anchor>
+            </li>
+            <li>
+              <Anchor
```
Comment (jeysal) [inline packages/react-components/src/molecules/CalendarLink.tsx:152]: I think this should be a `<button>` if it does not actually cause a navigation?

File: packages/react-components/src/molecules/CalendarLink.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  }, []);
+
+  useEffect(() => {
+    const handleClickOutside = (event: Event) => {
+      if (
+        reference &&
+        reference.current &&
+        !reference?.current?.contains(event.target as Node)
+      ) {
+        setMenuShown(false);
```
Comment (jeysal) [inline packages/react-components/src/molecules/CalendarLink.tsx:99]: This should be easily testable with `react-testing-library` - render `<><Component /><something-else /></>`, open the menu and click that something else to close it :sweat_smile:

File: packages/react-components/src/molecules/CalendarLink.tsx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+type CalendarLinkProps = {
+  readonly id: string;
+};
+
+const CalendarLink: React.FC<CalendarLinkProps> = ({ id }) => {
+  const reference = useRef<HTMLDivElement>(null);
+  const handleClick = () => setMenuShown(!menuShown);
+  const [menuShown, setMenuShown] = useState(false);
+  useEffect(() => {
```
Comment (jeysal) [inline packages/react-components/src/molecules/CalendarLink.tsx:88]: what is this for?

File: packages/react-components/src/molecules/CalendarLink.tsx [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+});
+
+const calendarLinkListStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  listStyle: 'none',
+  margin: 0,
+  padding: 0,
+
+  '& li': {
```
Comment (jeysal) [inline packages/react-components/src/molecules/CalendarLink.tsx:60]:
```suggestion
  '& > li': {
```
just to be safe, who knows what else we'll put somewhere deep down in there

File: packages/react-components/src/organisms/CalendarList.tsx [hunk 6]
```diff
@@ -100,19 +93,26 @@ const CalendarList: React.FC<CalendarListProps> = ({
...
-    )}
-  </Card>
+          ))}
+        </ul>
+      )}
+    </Card>
+    <Paragraph accent="lead">
+      Having issues? Set up your calendar manually with these instructions for{' '}
+      <Link href="https://support.apple.com/en-us/guide/calendar/icl1022/mac">
+        iCal
```
Comment (jeysal) [inline packages/react-components/src/organisms/CalendarList.tsx:107]: Umm, did apple seriously call their calendar `iCal`? I don't think so actually, isn't it `Apple Calendar` or so

**Review by jeysal — CHANGES_REQUESTED** [review]
I absolutely love the scammy buttons that all do the same thing :joy:

File: packages/react-components/src/organisms/CalendarList.tsx (hunk 6 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/CalendarList.tsx:107]: Yeah, they just call it Calendar.

File: packages/react-components/src/molecules/CalendarLink.tsx (hunk 4 above)
Comment (fampinheiro) [reply packages/react-components/src/molecules/CalendarLink.tsx:88]: :man_shrugging:

File: packages/react-components/src/organisms/CalendarList.tsx (hunk 6 above)
Comment (jeysal) [reply packages/react-components/src/organisms/CalendarList.tsx:107]: But...should we not ask design whether the copy should be changed then? :D

**Review by jeysal — APPROVED** [review]
LGTM if CI and patch cov becomes happy :sweat_smile:

File: packages/react-components/src/organisms/CalendarList.tsx (hunk 6 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/CalendarList.tsx:107]: Will they notice?

**Pull request merged**